src/core/tests/test_window.py

```diff
@@ -8,18 +6,15 @@
 
 
 class TestWindow(TestCase):
-
     def setUp(self):
         super().setUp()
         self.window = toga.Window(factory=toga_dummy.factory)
 
     def test_widget_created(self):
         id = self.window.id
         self.assertIsNotNone(self.window.id)
-        app = toga.App('test_name', 'id.app',
-                       factory=toga_dummy.factory)
-        new_app = toga.App('error_name', 'id.error',
-                           factory=toga_dummy.factory)
+        app = toga.App('test_name', 'id.app', factory=toga_dummy.factory)
+        new_app = toga.App('error_name', 'id.error', factory=toga_dummy.factory)
         self.window.app = app
         with self.assertRaises(Exception):
```

Comment:
Specifically what exception might this be expecting? I don't know how I feel about using a `Exception` here.